### PR TITLE
Capture memory also as part of monitor samples

### PR DESCRIPTION
**Pull request by puran157**
Capture Memory while logging samples.

Planning to edit https://github.com/ludeknovy/jtl-reporter/blob/main/scripts/jtl_listener_service.py and other files as well.
LMK if change looks good and then I can work on scripts.

### CODE REVIEW

**Comment by ludeknovy** [comment]
Hi @puran157 . Thanks for your contribution. I think in such a case it would be better to either create new API version, as the current change would break it for the older version ( you defined `mem` as required field) or the `mem` must be made optional.

**Comment by puran157** [comment]
> Hi @puran157 . Thanks for your contribution. I think in such a case it would be better to either create new API version, as the current change would break it for the older version ( you defined `mem` as required field) or the `mem` must be made optional.

Cool, let me make  it optional

**Comment by puran157** [comment]
> Hi @puran157 . Thanks for your contribution. I think in such a case it would be better to either create new API version, as the current change would break it for the older version ( you defined `mem` as required field) or the `mem` must be made optional.

Done, made mem optional

File: src/db/columnSets.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ export const SamplesColumnSet = new pg.helpers.ColumnSet([
 ], { table: new pg.helpers.TableName({ table: "samples", schema: "jtl" }) })
 
 export const MonitorColumnSet = new pg.helpers.ColumnSet([
-  "timestamp", "name", "cpu", {
+  "timestamp", "name", "cpu", "mem", {
```
Comment (ludeknovy) [inline src/db/columnSets.ts:58]: This is a breaking change, if "mem" wont be provided, the API will throw 500

**Review by ludeknovy — CHANGES_REQUESTED** [review]

File: src/db/columnSets.ts (hunk 1 above)
Comment (ludeknovy) [reply src/db/columnSets.ts:58]: I've created new MR with v4: https://github.com/ludeknovy/jtl-reporter-listener-service/pull/50

**Comment by ludeknovy** [comment]
Closing it, as https://github.com/ludeknovy/jtl-reporter-listener-service/pull/50 was merged. 
You can test it using the `latest` tag once this job is finished: https://github.com/ludeknovy/jtl-reporter-listener-service/actions/runs/9512898377

**Pull request closed without merging**